GrandCentralBoard/Widgets/Harvest/UIColor+BillingColor.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UIColor+BillingColor.swift
```

Comment:
Right now we have all widgets in one project, and if they all would name extensions on UIColor just `UIColor`, there would be a name clash.